config/routes.rb

```diff
@@ -84,12 +84,12 @@
       to: "organisations#court",
       as: :court
 
+  get "/world", to: "world#index"
+
   constraints DocumentTypeRoutingConstraint.new("step_by_step_nav") do
     get "/:slug", to: "step_nav#show"
   end
 
-  get "/world", to: "world#index"
-
```

Comment:
**question:** clarification

I'm not too sure what `DocumentTypeRoutingConstraint` does so I'm unclear what the need for this is, but it's not that important!

Reply:
From what I could understand, it's being used here to determine the controller and method using the returned document type (from within the content item), rather than by path.